### ISSUE TITLE
Feature request: Skip users you follow on another account

### ISSUE THREAD

**Issue by watatomo** [kheina-com/Blue-Blocker#242]
I have a separate account for following artists and publishers and whatnot. I figured I could try and gather them all and import them into a safelist, but that's over 4000 accounts and Musk made harder to get because you have to pay. 

On another note, it would be nice if there was a way to have the extension run only on tweets with replies. That's where the real annoyance is coming from (for me).

**Comment by cooljeanius** [comment]
related: issue #203

**Comment by watatomo** [comment]
Oh, that's exactly it. The title didn't catch my eye.

**Comment by cooljeanius** [comment]
> Oh, that's exactly it. The title didn't catch my eye.

Closing as a duplicate, then.

**Comment by rougetimelord** [comment]
I don't think that this is technically feasible unfortunately. The extension can't (currently) pull your full following list, it can only see if you follow the user when a tweet gets checked for premium subscriptions. To pull the full list, the extension would have to query the twitter API (which is almost definitely _possible_ but difficult). As a stop gap, the extension could add users it sees to the safelist but it wouldn't be 100% accurate.
